### ISSUE TITLE
Google Keep/Drive getting 404

### ISSUE THREAD

**Issue by Fooughhy** [getstation/desktop-app#393]
- Application version: 3.0.0
 - OS: Windows 10

**Steps to Reproduce:**

1. Open application
2. Add a new Google Keep/Drive application
3. Login to Google account (either choose an existing account, or enter a new one)
4. Once login is done, the application redirects to a page with HTTP Status Code 404.

The URL for the application becomes: https://drive.google.com/drive/u/<google-account-email>

That URL seems invalid as if I try it in my normal browser, I also get 404.

**Screenshots**
![image](https://github.com/getstation/desktop-app/assets/6977416/8e5f07bf-8d70-4899-83f7-f3767bbda432)

**Comment by viktor44** [comment]
Could you copy the URL of this 404 page?
Use Ctrl+Shift+U or menu item

**Comment by Fooughhy** [comment]
That's what I did, though I pasted it wrong I guess.
https://drive.google.com/drive/u/test@gmail.com

**Comment by ExploratorV** [comment]
I had this exact same issue and I was only able to solve it by uninstalling and reinstalling Station and resetting all the user settings. 

The only difference I could see is that there were two accounts displayed here when I was getting the error
![image](https://github.com/user-attachments/assets/e0a3b41a-b45e-4f86-a1c1-73c92a95e115)

The first one was my work account that I never intended using in station so I deleted the associated app, but it kept appearing on this screen. Not sure if it's related, but now that my intended account is the first and only one, it works fine.

**Comment by Torjas** [comment]
Yes, I also have this Problem when I add multiple accounts. The Mail address should probably be replaced with the index of the account in station to make it work
